skipruntime-ts/Makefile

```
@@ -4,7 +4,7 @@ SKARGO_PROFILE?=release
 
 .PHONY: build
 build:
-	../bin/cd_sh .. "npm install && npm run build"
+	../bin/cd_sh .. "SKIPRUNTIME=$$(pwd)/../build/skipruntime npm install && npm run build"
```

I haven't tested, but I expect that expanding the CURDIR make variable before running the shell command does the right thing
```suggestion
	../bin/cd_sh .. "SKIPRUNTIME=$(CURDIR)/build/skipruntime npm install && npm run build"
```